sdk/appconfiguration/azure-appconfiguration/azure/appconfiguration/_generated/_patch.py

```diff
@@ -6,9 +6,119 @@
 
 Follow our quickstart for examples: https://aka.ms/azsdk/python/dpcodegen/python/customize
 """
-from typing import List
+from typing import List, Optional, Any, Union
+from azure.core import PipelineClient
+from azure.core.credentials import AzureKeyCredential, TokenCredential
+from azure.core.pipeline import policies
 
-__all__: List[str] = []  # Add all objects you want publicly available to users at this package level
+from azure.appconfiguration._generated import models as _models
```

Comment:
since you're already in generated, you should just be able to do local imports,
i.e.
```python
from . import models as _models
from ._azure_app_configuration import ...
```

Reply:
Updated